boards/x86/up_squared/Kconfig.defconfig

```diff
@@ -11,4 +11,7 @@ config BUILD_OUTPUT_STRIPPED
 config APIC_TIMER
 	default y if !HPET_TIMER
 
+config X86_MMU_PAGE_POOL_PAGES
+	default 3072 if X86_MMU
```

Comment:
When I boot my Up Squared I get: `Optimal CONFIG_X86_MMU_PAGE_POOL_PAGES 1033`

Reply:
With this PR, that is

Reply:
How much memory does you board have? I think this depends on the configuration, 2GB vs 4GB

Reply:
This number depends on how much memory is marked by the MMU code, which is dependent on the app size. A large number should be enough to cover most apps.